### PR TITLE
i18n(es): Update `images` `V3`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Update `images` `V3` [`PR`](https://github.com/withastro/docs/pull/3739/files#diff-9ea5d63bc29a5a013378b352f2b77fffd61f773692ff6c329a3cc9292552ac23)

### CODE REVIEW

File: src/content/docs/es/guides/images.mdx [hunk 1]
```diff
@@ -4,152 +4,703 @@ description: Aprende cómo usar imágenes en Astro.
...
+  height="900"
+  decoding="async"
+  loading="lazy"
+  alt="Una descripción de mi imagen."
+/>
+```
+
+Actualmente, la característica integrada de assets no incluye un componente `<Picture />`.
+
+En su lugar, puedes [generar imágenes o componentes personalizados usando `getImage()`](#generando-imágenes-con-getImage) que utilicen los atributos HTML de imagen `srcset` y `sizes` o la etiqueta `<picture>` [para la dirección de arte o para crear imágenes responsivas](https://developer.mozilla.org/es/docs/Aprender/HTML/Multimedia_y_incrustaciones/Im%C3%A1genes_responsivas#direcci%C3%B3n_de_arte).
```
Comment (Waxer59) [inline src/content/docs/es/guides/images.mdx:88]:
```suggestion
En su lugar, puedes [generar imágenes o componentes personalizados usando `getImage()`](#generando-imágenes-con-getImage) que utilicen los atributos HTML de imagen `srcset` y `sizes` o la etiqueta `<picture>` [para la dirección de arte o para crear imágenes responsivas](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction).
```

File: src/content/docs/es/guides/images.mdx [hunk 2]
```diff
@@ -4,152 +4,703 @@ description: Aprende cómo usar imágenes en Astro.
...
-<img src="https://docs.astro.build/assets/logomark-light.png" width="25" alt="Astro" />
+<Image src="https://example.com/images/remote-image.png" />
+<img src="https://example.com/images/remote-image.png" />
+![Astro](https://example.com/images/remote-image.png)
+
+```
+
+## Imágenes en Colecciones de Contenido
+
+You can declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder:
```
Comment (dreyfus92) [inline src/content/docs/es/guides/images.mdx:366]: Falta traducir.

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo una sugerencia 😁

File: src/content/docs/es/guides/images.mdx (hunk 2 above)
Comment (Waxer59) [inline src/content/docs/es/guides/images.mdx:366]:
```suggestion
Puedes declarar una imagen asociada para una entrada de colecciones de contenido, como la imagen de portada de una entrada de blog, en tu metadatos utilizando su ruta relativa a la carpeta actual:
```

**Comment by Waxer59** [comment]
Listo!, gracias por la revisión 🤠

**Comment by sarah11918** [comment]
Looking great @Waxer59 & @dreyfus92 ! Please see the last minute change I made to the images page, 4. Removing Sharp in the main PR, and then I think we're calling the Image PR done!  :partying_face:

File: src/content/docs/es/guides/images.mdx [hunk 3]
```diff
@@ -4,152 +4,703 @@ description: Aprende cómo usar imágenes en Astro.
...
+    Actualmente, la función de assets integrada no incluye un componente `<Picture />`.
 
-Para encontrar la documentación de como usar `@astrojs/image` en Astro v2, por favor ve a la [documentación del paquete `@astrojs/image`](https://github.com/withastro/astro/blob/main/packages/integrations/image/README.md)
+    En su lugar, puedes utilizar los atributos de imagen HTML `srcset` y `sizes`, o la etiqueta `<picture>` [para dirección de arte o para crear imágenes responsivas](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction).
 
-## Usar imágenes de un CMS o una CDN
+4. Elimina Sharp
 
-Las CDNs de imágenes funcionan con Astro. Utiliza la URL completa de una imagen como atributo `src` en una etiqueta `<img>` o notación de Markdown.
+    [Sharp](https://github.com/lovell/sharp) es ahora el servicio de imágenes por defecto utilizado para `astro:assets`. Ya no necesitas instalar la dependencia localmente en tu proyecto.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/images.mdx:686]: Cambia esto con esto a0bc8c528a725775caa452368185f12e3e90a825

**Comment by Yan-Thomas** [comment]
> Did I miss any other commit?

I believe not! Thanks @Waxer59 🙌

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by kevinzunigacuellar — APPROVED** [review]
LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**